export_unity_package_config.json

```diff
@@ -5,37 +5,71 @@
       "imports": [
         {
           "importer": "PluginImporter",
-          "platforms": ["Editor"],
+          "platforms": [
+            "Editor"
+          ],
           "paths": [
             "ExternalDependencyManager/Editor/Google.VersionHandler.*"
           ]
         },
         {
           "importer": "PluginImporter",
           "platforms": [],
-          "labels": ["gvhp_targets-editor"],
+          "labels": [
+            "gvhp_targets-editor"
+          ],
           "paths": [
-            "ExternalDependencyManager/Editor/*/Google.IOSResolver.*",
             "ExternalDependencyManager/Editor/*/Google.JarResolver.*",
             "ExternalDependencyManager/Editor/*/Google.VersionHandlerImpl.*",
             "ExternalDependencyManager/Editor/*/Google.PackageManagerResolver.*"
           ],
           "override_metadata_upm": {
             "PluginImporter": {
-              "platformData": [ {
-                  "first" : {
-                      "Editor": "Editor"
+              "defineConstraints": [
+                "UNITY_EDITOR",
+                "UNITY_IOS"
```

Comment:
Unless I'm misunderstanding how the package config works, shouldn't be in the new section below? This currently looks like it affects:
- `Google.JarResolver.*`
- `Google.VersionHandlerImpl.*`
- `Google.PackageManagerResolver.*`

And _not_:
- `Google.IOSResolver.*`

Reply:
> Unless I'm misunderstanding how the package config works, shouldn't be in the new section below? 

Yes it is now split out into its own section.